doc/release-notes/6.1-release-notes.md

```diff
@@ -0,0 +1,303 @@
+# Dataverse 6.1
+
+Please note: To read these instructions in full, please go to https://github.com/IQSS/dataverse/releases/tag/v6.1 rather than the list of releases, which will cut them off.
+
+This release brings new features, enhancements, and bug fixes to the Dataverse software.
+Thank you to all of the community members who contributed code, suggestions, bug reports, and other assistance across the project.
+
+## Release Highlights (Major Upgrades, Breaking Changes)
+
+This release contains major upgrades to core components. Detailed upgrade instructions can be found below.
+
+## Detailed Release Highlights, New Features and Use Case Scenarios
```

Comment:
Should these headings be at the same level (`##`)? Or should we have a single heading at this level? Seems odd.

Reply:
removed second header. I don't believe it adds anything